lib/vector-core/src/event/vrl_target.rs

```diff
@@ -411,6 +388,111 @@ fn target_get_metric(path: &LookupBuf, metric: &Metric) -> Result<Option<Value>,
     Ok(None)
 }
 
+/// pre-compute the `Value` structure of the metric.
```

Comment:
```suggestion
/// Pre-compute the `Value` structure of the metric.
```

Reply:
Resolved in [f24dfb7](https://github.com/vectordotdev/vector/pull/12546/commits/f24dfb7427b02d367905ab00817a783de1c2a2bb).